resources/js/requests/components/FilterMobile.vue

```diff
@@ -0,0 +1,217 @@
+              class="btn btn-secondary dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="fas fa-circle text-warning"></i>
+              {{ selectedOption }}
+              <i class="fas fa-caret-down"></i>
+            </button>
+            <div
+              class="dropdown-menu"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('In Progress', 'status')"
+                ><i class="fas fa-circle text-warning"></i> In Progress</a
+              >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('Completed', 'status')"
+                ><i class="fas fa-circle text-primary"></i> Completed</a
+              >
+            </div>
+          </div>
+
+          <div
+            class="dropdown"
+            v-if="type === 'requests'"
+          >
+            <button
+              class="btn btn-secondary dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="fas fa-user"></i>
+              <i class="fas fa-caret-down"></i>
+            </button>
+            <div
+              class="dropdown-menu"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('Requested by Me', 'filter')"
+                ><i class="fas fa-user"></i> Requested by Me</a
+              >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('With me as Participant', 'filter')"
+                ><i class="fas fa-users"></i> With me as Participant</a
+              >
+            </div>
+          </div>
+
+          <div
+            class="dropdown"
+            v-if="type === 'tasks'"
+          >
+            <button
+              class="btn btn-secondary dropdown-toggle"
+              type="button"
+              id="dropdownMenuButton"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <i class="fas fa-list"></i>
+              <i class="fas fa-caret-down"></i>
+            </button>
+            <div
+              class="dropdown-menu"
+              aria-labelledby="dropdownMenuButton"
+            >
+              <a
+                class="dropdown-item"
+                href="#"
+                @click="selectOption('By Due Date', 'orderBy')"
+              >
+                By Due Date</a
```

Comment:
??

Reply:
By Due Date is a new Order by criteria requested in Mockups.